src/main/java/co/elastic/maven/opentelemetry/OtelExecutionListener.java

```diff
@@ -85,6 +86,7 @@ public String get(@Nullable Map<String, String> environmentVariables, String key
                     .setAttribute(MavenOtelSemanticAttributes.MAVEN_PROJECT_GROUP_ID, project.getGroupId())
                     .setAttribute(MavenOtelSemanticAttributes.MAVEN_PROJECT_ARTIFACT_ID, project.getArtifactId())
                     .setAttribute(MavenOtelSemanticAttributes.MAVEN_PROJECT_VERSION, project.getVersion())
+                    .setSpanKind(SpanKind.SERVER)
```

Comment:
I don't know if `sessionStarted` applies once even for multi-module maven projects.